README.md

```diff
@@ -68,6 +68,8 @@ This Puppet module provides secure configuration of your base OS with hardening.
   the number of last passwords (e.g. 5 will prevent user to reuse any of her last 5 passwords)
 * `allow_change_user = false`
   if a user may use `su` to change his login
+* `only_root_may_su = true`
```

Comment:
I would better set the default to `false`, because this change might otherwise break existing installations